spec/cypress/app_commands/scenarios/admin/school_participants.rb

```diff
@@ -44,3 +44,10 @@
                   user: FactoryBot.create(:user, full_name: "Unrelated ect"),
                   school_cohort: another_school_cohort,
                   created_at: Date.parse("29/12/2020")
+
+npq_profile = FactoryBot.create :participant_profile, :npq, validation_data: nil
+
+FactoryBot.create :npq_validation_data,
+                  id: npq_profile.id,
+                  user: FactoryBot.create(:user, full_name: "Natalie Portman Quebec"),
+                  created_at: Date.parse("19/09/2019")
```

Comment:
Do we need to pin date of birth to avoid randomness in percy?

Reply:
Yup!